bpf/lib/conntrack.h

```diff
@@ -320,7 +321,11 @@ __ct_lookup(const void *map, struct __ctx_buff *ctx, const void *tuple,
 				entry->seen_non_syn = false;
 
 				*monitor = ct_update_timeout(entry, is_tcp, dir, seen_flags);
-				return CT_REOPENED;
+
+				/* Return CT_NEW so that the caller creates a new entry instead of
+				 * updating the old one. (For policy drops the old entry remains.)
+				 */
+				return CT_NEW;
```

Comment:
Do you think we should remove the `ct_reset_closing()` and `ct_reset_seen_flags()` here as well? I'm undecided on the `ct_update_timeout()` call ... it almost feels like we should just have a `goto ct_new`.

Reply:
Not sure. It is still possible that a new entry will not be created due to a policy DROP, so I left these be.

Reply:
Hmm yes makes sense.